### ISSUE TITLE
Are we going to see another update at some point?

### ISSUE THREAD

**Issue by Zeblote** [NVIDIAGameWorks/PhysX#309]
Hey, so it's been almost a year since PhysX 4.1, and complete radio silence since then. I'm sure lots of optimizations and bug fixes have added up over this time. Wouldn't it be nice to see another update?

**Comment by Hoikas** [comment]
It would be nice if pull requests that have been accepted into NVIDIA's internal main were also merged here on GitHub. As it stands, they are shunted off into the "closed" tab where it is difficult to find them.

**Comment by Zeblote** [comment]
https://twitter.com/PierreTerdiman/status/1284124994751037442 PhysX 5.1 seems to be a thing already?

**Comment by amoravanszky** [comment]
The PhysX 5.1 SDK is not yet released, it is under development internally, and Pierre is clearly too excited to keep quiet about it.  

PhysX 5 however is already being used in NVIDIA Omniverse, and in particular it is going out to all developers who have registered to to access Isaac Sim: https://developer.nvidia.com/isaac-sim

We do plan to update this github with a source release eventually, but I don't have a date for when that will happen.  Since we are now developing a full stack of our own tooling to go on top of the SDK, as well as features well beyond just rigid bodies, finishing it is taking longer than in the past when the scope was limited to a game physics runtime library.

**Comment by Zeblote** [comment]
Hi @amoravanszky, thanks for responding! Am I understanding it correctly that registering to access Isaac Sim does **not** give access to the PhysX 5 SDK (with a license to redistribute the binaries)?

**Comment by amoravanszky** [comment]
Correct, you can't develop your own software using it, it is only for using as part of Isaac Sim.  So, yes, a C++ software development kit of PhysX 5 for making your own software is not yet available.

**Comment by Zeblote** [comment]
Any news though? 2020 is going to be over soon!

**Comment by ghost** [comment]
I'm also curious tbh

**Comment by ghost** [comment]
Would it be reasonable to guess that it might be 2021 source release instead of 2020 due to it being end of the year?

**Comment by Zeblote** [comment]
Maybe 2022 is more likely?

**Comment by dyanikoglu** [comment]
Maybe 2023? 🙄

**Comment by bryansee** [comment]
Maybe sometime after the end of the coronavirus pandemic, which happens at the heat death of the universe or the end of time, perhaps?

**Comment by Zeblote** [comment]
3 months to go until the first anniversary of this issue!

**Comment by dyanikoglu** [comment]
Unreal's Chaos becoming stable vs. Nvidia releasing PhysX 5. Who would win? :)

**Comment by Nulled-Out** [comment]
> The PhysX 5.1 SDK is not yet released, it is under development internally, and Pierre is clearly too excited to keep quiet about it.  
> 
> PhysX 5 however is already being used in NVIDIA Omniverse, and in particular it is going out to all developers who have registered to to access Isaac Sim: https://developer.nvidia.com/isaac-sim
> 
> We do plan to update this github with a source release eventually, but I don't have a date for when that will happen.  Since we are now developing a full stack of our own tooling to go on top of the SDK, as well as features well beyond just rigid bodies, finishing it is taking longer than in the past when the scope was limited to a game physics runtime library.  
 
---
 
I know recent events could've delayed things but was previewed in 2019 & it's almost 2022 now. 

Given that it's shown to be complete enough in Omniverse, would it not be fair for those who were led to believe that they could depend on that release to atleast get a public announcement saying it's not coming?  @amoravanszky as opposed to people searching and seeing the articles that PhysX5 with DEM would be available as Open-Source? 


Surely you can understand the frustrations & disappointment people may feel after plans would've been made or whatever else would have been impacted by the radio-silence and feeling as if it's been brushed away and left out to dry 🙁
